templates/vhost/_ssl.erb

```diff
@@ -19,23 +30,12 @@
   <%- if @ssl_crl -%>
   SSLCARevocationFile     "<%= @ssl_crl %>"
   <%- end -%>
+  <%- if @ssl_verify_depth -%>
+  SSLVerifyDepth          <%= @ssl_verify_depth %>
```

Comment:
what was the purpose of changing the order of these directives?

Reply:
I was trying to regroup the directives that have no effect when client certs are not being looked at/needed... I just moved them all around into one block to make it easier to read (and missed that one the first time around).

Reply:
it's not as clear as i'd like from the diff...
the set of directives that i moved in here are only effective is "ssl_verify_client" on line 19 is set. That "if" statement ends on line 39. That means that all the directives that are "ssl client auth specific" are only included in the config when they are actually needed/used.

This might be out of scope from the pull request perspective, but will make it clear that those are only to be used when the setup enforce verification of client certificates for authentication.